src/Content/Text/BBCode.php

```diff
@@ -1361,6 +1361,12 @@ function ($match) {
 				. '</a>';
 		}, $text);
 
+		// We need no target="_blank" for local links
+		// convert links start with System::baseUrl() as local link without the target="_blank" attribute
+		$escapedBaseUrl = preg_quote(System::baseUrl(), '/');
+		$text = preg_replace("/\[url\]($escapedBaseUrl)([$URLSearchString]*)\[\/url\]/ism", '<a href="$1$2">$1$2</a>', $text);
+		$text = preg_replace("/\[url\=($escapedBaseUrl)([$URLSearchString]*)\](.*?)\[\/url\]/ism", '<a href="$1$2">$3</a>', $text);		
```

Comment:
```suggestion
		$text = preg_replace("/\[url\=($escapedBaseUrl[$URLSearchString]*)\](.*?)\[\/url\]/ism", '<a href="$1">$2</a>', $text);		
```

Reply:
Had to quote out the ```$escapedBaseUrl``` otherwise the ```[$URLSearchString]``` was seen as string index ;)